src/contacts/Main.java

```diff
@@ -0,0 +1,73 @@
+package contacts;
+
+
+import contacts.processors.*;
+import contacts.processors.impl.*;
+import contacts.service.IRecordService;
+import contacts.service.RecordServiceImpl;
+
+import java.io.IOException;
+import java.util.InputMismatchException;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
+            IRecordService recordService = new RecordServiceImpl();
+
+            IOrganizationActionProcessor actionProcessorEditAddressOrganization = new EditOrganizationAddressProcessor(recordService);
```

Comment:
Please, rename all of objects according to pattern editAddressOrganizationProcessor

Reply:
Done